contractcourt/channel_arbitrator_test.go

```diff
@@ -62,6 +62,34 @@ func (b *mockArbitratorLog) FetchUnresolvedContracts() ([]ContractResolver,
 	for resolver := range b.resolvers {
 		v[idx] = resolver
 		idx++
+
+		// In order to permit restarts, for all possible resolvers,
+		// we'll give them a fresh quit channel. This is required as
+		// otherwise, if a restart is simulated within a test (stopping
+		// the existing channel arb along the way), then the resolvers
+		// will immediately exit, as their quit channel is still
+		// closed.
+		switch res := resolver.(type) {
+		case *htlcSuccessResolver:
+			res.Quit = make(chan struct{})
+			res.htlcAmt = 0
+
+		case *htlcIncomingContestResolver:
+			res.Quit = make(chan struct{})
+			res.htlcAmt = 0
+			res.circuitKey = channeldb.CircuitKey{}
+
+		case *htlcOutgoingContestResolver:
+			res.Quit = make(chan struct{})
+			res.htlcAmt = 0
+
+		case *htlcTimeoutResolver:
+			res.Quit = make(chan struct{})
+			res.htlcAmt = 0
+
+		case *commitSweepResolver:
+			res.Quit = make(chan struct{})
+		}
```

Comment:
This block can be removed now?

Reply:
Hmm, must've slipped in a rebase, removed!